data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -0,0 +1,951 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# + tags=["parameters"]
+project_id = ""
+old_rdr = ""
+new_rdr = ""
+raw_rdr = "default"  # do not need to provide this if running on a raw rdr import
```

Comment:
I believe this parameter is not needed. There would not be a rdr sandbox at this stage and the queries that use this parameter should be performed after cleaning. I'll mark them below.